CHANGELOG.md

```diff
@@ -2,15 +2,34 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.
 
+2.20.0 (In Progress)
+====================
+
+Changes
+-------
+
+The [Video.runPreflight](https://sdk.twilio.com/js/video/releases/2.20.0/docs/module-twilio-video.html#.runPreflight__anchor) API has been promoted to GA. Below are the changes included in this release. (VIDEO-7728)
```

Comment:
Maybe for this intro we can link to 2.16.0 since it has more information + give folks from the community a shoutout: 

The Preflight API ([runPreflight](https://sdk.twilio.com/js/video/releases/2.20.0/docs/module-twilio-video.html#.runPreflight__anchor)), originally released in [2.16.0](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md#2160-august-11-2021), has been promoted to GA. 

Thank you @ morninng @ eroidaaruqaj [#1622](https://github.com/twilio/twilio-video.js/issues/1622) for your feedback. Based on this feedback, we have made the following changes to `runPreflight`. (VIDEO-7728)